packages/smpc/test_fpt.py

```diff
@@ -13,3 +13,11 @@
 print("=====")
 value = fpt_tensor.decode()
 print(value)
+
+
+val = Tensor(orig_value)
+fpt_val = val.fix_precision()
```

Comment:
great job with this - looks good

Reply:
Not sure about folder construction here but if you feel comfortable with it we can make it work. In v1 it's likely that PySyft will hardcode itself to the permissions infra we setup here. Sure it would be nice to have it be in separate packages but we've got so much ground to cover I'm hesitant to do a lot of packaging stuff right now. (aka to be putting sympc in a separate package as opposed to just making a sympc module)